tickets/src/components/content/EventVerify.js

```diff
@@ -51,7 +67,13 @@ export const mapDispatchToProps = dispatch => ({
 })
 
 export const mapStateToProps = ({ router, locks, account, event }) => {
-  const { lockAddress } = lockRoute(router.location.pathname)
```

Comment:
This is a rather brittle change, can we instead update the regexp and `lockRoute`?